notification-scripts/notifyV2

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env sh
```

Comment:
How will this script behave in the absence of `jq`? Will it fail gracefully?

Reply:
Resolved by commit 60f4c041 .

Reply:
Resolved by commit 60f4c04 .